sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -955,11 +955,13 @@ class SGDClassifier(BaseSGDClassifier):
         value, the stronger the regularization.
         Also used to compute the learning rate when set to `learning_rate` is
         set to 'optimal'.
+        Values should be in the range `[0.0, inf)`.
 
     l1_ratio : float, default=0.15
         The Elastic Net mixing parameter, with 0 <= l1_ratio <= 1.
         l1_ratio=0 corresponds to L2 penalty, l1_ratio=1 to L1.
         Only used if `penalty` is 'elasticnet'.
+        Values should be in the range `[0.0, 1.0]`.
```

Comment:
This is redundant with the first line of this parameter's docstring paragraph.

No strong opinion on this change. I am fine with either keeping it or removing it for conciseness.

Reply:
Let's keep as is, for consistency.